charts/incubator/tasmoadmin/values.yaml

```diff
@@ -15,8 +15,14 @@ workload:
     podSpec:
       containers:
         main:
+          probes:
+            liveness:
+              type: http
+            readiness:
+              type: http
+            startup:
+              type: http
```

Comment:
Why are these here?
@xstar97 bad advice, probes for main service are **ALWAYS** configured.
It also has nothing to do with this PR.